shared_model/builders/protobuf/unsigned_proto.hpp

```diff
@@ -63,7 +66,7 @@ namespace shared_model {
         if (boost::size(object_.signatures()) == 0) {
           throw std::invalid_argument("Cannot get object without signatures");
         }
-        return object_;
+        return std::move(object_);
```

Comment:
I am not sure about the safety of this method because its class may be in bindings. So, here we have to care about `signAndAddSignature` method and check that this method works correctly.